src/app/features/dashboard/dashboard.component.ts

```diff
@@ -52,4 +52,8 @@ export class DashboardComponent implements OnInit, OnDestroy {
     this.cryptoItems[item.id] = item;
   }
 
+  removeItem(item: CryptoItem) {
+    this.cryptoItems = this.cryptoItems.filter(ele => { return ele.id != item.id });
```

Comment:
LGTM :+1:  Yust one thing, can you rename `ele` to the common names `el`, `elem` or `element`?